src/saleor_app/deps.py

```diff
@@ -43,39 +44,72 @@ async def saleor_token(
     return token
 
 
+async def saleor_app_token(
```

Comment:
Couldn't we have 1 fn isntaead of 2 (saleor_token, saleor_app_token) and simply use different params?